beta/src/components/MDX/Sandpack/useSandpackLint.tsx

```diff
@@ -29,7 +29,9 @@ export const useSandpackLint = () => {
         // Ignore parsing or internal linter errors.
         const isReactRuleError = (error: any) => error.ruleId != null;
         setLintErrors(errors.filter(isReactRuleError));
-        return codeMirrorErrors.filter(isReactRuleError);
+        return codeMirrorErrors
+          .filter(isReactRuleError)
+          .map((error) => ({...error, severity: 'error'}));
```

Comment:
I could update it directly in the `runEslint` file, but I'd do all the customizations in the UI layer and leave the eslint integration as pure as possible.